Gemfile

```diff
@@ -31,6 +32,7 @@ gem 'turbolinks', github: 'rails/turbolinks'
 gem 'jquery-turbolinks'
 gem 'rails-timeago'
 gem 'whenever', :require => false
+gem 'sidekiq'
```

Comment:
Let's use https://github.com/chanks/que to avoid additional dependency on redis.